src/components/CreateCoursePage/index.jsx

```diff
@@ -183,9 +166,11 @@ class CreateCoursePage extends React.Component {
                 initialValues={initialValues}
                 currentFormValues={formValues}
                 organizations={organizations}
+                // TODO: Add in logic here to decide when course type is being used
```

Comment:
Nice. I 100% agree with your delaying this decision. But I do like the idea of like "eh... we should like, add logic maybe?"